exonum/src/storage/error.rs

```diff
@@ -14,11 +14,9 @@
 
 //! An implementation of `Error` type.
 
-use std::fmt;
-use std::error;
-
 /// The error type for I/O operations with storage.
-#[derive(Debug, Clone)]
```

Comment:
Is there a motivation behind removing `Clone`?

Reply:
No, Clone is back!